dcos/cosmospackage.py

```diff
@@ -155,16 +155,20 @@ def installed_apps(self, package_name, app_id):
 
         return packages
 
-    def get_repos(self):
+    def get_repos(self, json_):
         """List locations of repos
 
-        :returns: the list of repos, in resolution order
-        :rtype: [str]
+        :param json_: output json if True
+        :type json_: bool
+        :returns: the list of repos, in resolution order or list
+        :rtype: [str] | dict
```

Comment:
Not a huge fan of this. Why not always return the `dict` do the formatting in the caller?